pkgs/clightning-plugins/default.nix

```
@@ -3,25 +3,14 @@ pkgs: nbPython3Packages:
 let
   inherit (pkgs) lib;
 
-  src = pkgs.applyPatches {
-    src = pkgs.fetchFromGitHub {
+  src = pkgs.fetchFromGitHub {
       owner = "lightningd";
       repo = "plugins";
```

Can you fix the indentation to use two spaces?
```suggestion
    owner = "lightningd";
    repo = "plugins";
    ...
```